docs/user/monitor_experiments_with_tensorboard.md

```diff
@@ -0,0 +1,178 @@
+
+First, add TensorBoard output type to `logger`. Pass `log_dir` as the save location of the TensorBoard event files :
+```bash
+logger.add_output(dowel.TensorBoardOutput(log_dir))
+```
+
+### Log **Scalar** Values to TensorBoard
+
+You would need [`dowel.TabularInput`](https://github.com/rlworkgroup/dowel/blob/master/src/dowel/tabular_input.py).
+
+To prepare your TensorBoard output, import dowel's TabularInput instance `tabular` by:
+
+`from dowel import tabular`
+
+Add scalar values:
+```bash
+tabular.record('Epoch', epoch)
+tabular.record('# Sample', i_sample)
+tabular.record('AverageDiscountedReturn', return)
+```
+
+Finally, log your scalar values with `logger`:
+
+`logger.log(tabular)`
+
+### Log **histograms** to Tensorboard
+
+To log histograms to TensorBoard, you would need to pass [`dowel.Histogram`](https://github.com/rlworkgroup/dowel/blob/master/src/dowel/histogram.py) to `tabular`.
+
+`dowel.Histogram` is implemented as a typed view of a numpy array. It will accept input that `numpy.asarray` will.
+
+For example:
+```bash
+from dowel import Histogram, logger, tabular
+
+samples = norm.rvs(100)  # ndarray of 100 samples of a distribution
+hist = Histogram(samples)
+tabular.record('Sample', hist)
+
+logger.log(tabular)
+```
+
+## The "Automatic" Way: Setup Your Experiment with `@wrap_experiment`
+Garage provides a decorator `@wrap_experiment` to help the logger setup process.
+
+Suppose you have an experiment function `my_experiment()` with the following definition:
+```bash
+def my_experiment(seed=1):
+```
+Here's how you can enable logging for this experiment:
+
+First, import the `wrap_experiment` decorator function.
+```bash
+from garage import wrap_experiment
+```
+
+Now wrap your function:
+```bash
+@wrap_experiment
+def my_experiment(ctxt=None, seed=1):
+    """
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+    """
+```
+`@wrap_experiment` helps create experiment log directories and set up logger outputs (e.g. `logger.add_output(dowel.TensorBoardOutput(log_dir))`). After the setup process, `my_experiment()` is called to start the actual experiment.
+
+Note that `my_experiment()` now receives a new input `ctxt`. This is required for  the wrapped function to receive experiment configuration created by the wrapper.
+
+This decorater can also be invoked **with arguments**:
+
+```bash
+@wrap_experiment(log_dir='my_dir')
+def my_experiment(ctxt, seed):
+```
+For a complete list of arguments, you can refer to the definition of `wrap_experiment` [here](https://github.com/rlworkgroup/garage/blob/master/src/garage/experiment/experiment.py).
+
+Congrats! Now you have finished setting up TensorBoard logging. You can run your experiment and find your TensorBoard logs under the log directory.
+
+By default, your `log_dir` is `garage/data/local/experiment/{experiment_name}_{experiment_id}`
```

Comment:
logs actually go to `$(pwd)/data/local` and remember that most users are not downloading the repo (they install from pip), so the prefix `garage/` makes no sense.